src/ray/raylet/worker_pool.cc

```diff
@@ -89,10 +89,10 @@ WorkerPool::WorkerPool(instrumented_io_context &io_service,
                        const std::vector<int> &worker_ports,
                        std::shared_ptr<gcs::GcsClient> gcs_client,
                        const WorkerCommandMap &worker_commands,
-                       const std::string &native_library_path,
+                       std::string native_library_path,
                        std::function<void()> starting_worker_timeout_callback,
                        int ray_debugger_external,
-                       const std::function<double()> get_time)
```

Comment:
can we do more by changing this func to a `std::function<absl::Time()>`?

Reply:
Yes, but that's a much bigger change...

Reply:
ok we can do that in another PR, some time later...

Reply:
Ok I updated to use `absl::Time` everywhere